### PR TITLE
Create ipswdownloader.rb

### PR DESCRIPTION
**Pull request by thibmaek**

### CODE REVIEW

File: Casks/ipswdownloader.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'ipswdownloader' do
+  if MacOS.release < :lion
```
Comment (vitorgalvao) [inline Casks/ipswdownloader.rb]: Should be inclusive (i.e. `<= snow_leopard`).

File: Casks/ipswdownloader.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    version '1.6.0'
+    sha256 '82a236f69d2b7acfa9b0c801b9df76372005d5aab3f399f0d10db71320bb0af3'
+    url "http://www.igrsoft.com/wp-content/plugins/download-monitor/download.php?id=48"
```
Comment (vitorgalvao) [inline Casks/ipswdownloader.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

File: Casks/ipswdownloader.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  else
+    version '2.6.4'
+    sha256 'ba94d309c8c52b1bf2e58e2afc18eadb1a753e39ce180a255369c13a723e6018'
+    url "http://www.igrsoft.com/wp-content/plugins/download-monitor/download.php?id=69"
```
Comment (vitorgalvao) [inline Casks/ipswdownloader.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

Comment (thibmaek) [inline Casks/ipswdownloader.rb]: All changed in most recent commit.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by thibmaek** [comment]
Hmm seems there's no way to do that from Github directly, I'll just refork the current repo and create a correct PR this time.

**Pull request closed without merging**